app/services/menu/update.rb

```diff
@@ -0,0 +1,19 @@
+class Menu
+  class Update < ApplicationService
+    def initialize(params = {})
+      @id = params[:id]
+    end
+
+    def call
+      menu if menu.save
+    end
```

Comment:
Why do you need this service?
It does literally nothing except of useless SQL query